static/app/views/settings/projectMetrics/metricsExtractionRuleForm.tsx

```diff
@@ -555,6 +606,29 @@ export function MetricsExtractionRuleForm({
   );
 }
 
+function MenuList({
+  children,
+  info,
+  ...props
+}: React.ComponentProps<typeof components.MenuList> & {info: React.ReactNode}) {
+  const theme = useTheme();
+  return (
+    <components.MenuList {...props}>
+      <div
+        css={css`
+          /* The padding must align with the values specified for the option in the forms/controls/selectOption component */
+          padding: ${space(1)};
+          padding-left: calc(${space(0.5)} + ${space(1.5)});
```

Comment:
Is there a reason why this has to be done like this?

Reply:
I could use `${space(2)}`, but I prefer to be more explicit about the value. However, it might not be clear enough

![Screenshot 2024-08-01 at 09 34 30](https://github.com/user-attachments/assets/cf15df5d-1ddf-476b-be35-10fe33b67a43)
![Screenshot 2024-08-01 at 09 34 37](https://github.com/user-attachments/assets/96531819-c167-41e4-88bf-bb8bedd86655)

Reply:
the options are wrapped with elements that have different paddings. 1 element has `${space(0.5)} ` and the other `${space(1.5)}`

Reply:
Ok, got it, makes sense then